### PR TITLE
docs(components): Convert docs files

### PR DESCRIPTION
**Pull request by taylorvnoj**
### Added

Guides:
- Create a React Component
- Frontend styleguide
- Getting started with React
- Pull request title generator 

Content:
- Formatting
- Product vocabulary
- Voice & tone

Misc:
- Introduction (README)

### Decisions
- Put the README as Introduction to match `jobber-mobile` storybook

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/getting-started-with-react/getting-started-with-react.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+## Composing Components
+
+As mentioned above a lot of the power of React comes from being able to build
+components out of other components. Lets take a look at an example on how to do
+that.
+
+For this example we are going to build a card that allows you to enter a topic
+and then search for a related gif.
+
+export function ComposingComponents() {
```
Comment (MichaelParadis) [inline docs/getting-started-with-react/getting-started-with-react.stories.mdx:101]: Could this be an example with hooks?

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Minor comment. Let me know if you have any questions

File: docs/getting-started-with-react/getting-started-with-react.stories.mdx (hunk 1 above)
Comment (taylorvnoj) [reply docs/getting-started-with-react/getting-started-with-react.stories.mdx]: Yes it can! Good catch - thank you!

**Review by MichaelParadis — APPROVED** [review]
LGTM!

**Pull request merged**